jwst/stpipe/integration.py

```diff
@@ -0,0 +1,88 @@
+"""
+Entry point implementations.
+"""
+
+
+def get_steps():
+    """
+    Return tuples describing the stpipe.Step subclasses provided
+    by this package.  This method is registered with the stpipe.steps
+    entry point.
+
+    Returns
+    -------
+    list of tuple (str, str, bool)
```

Comment:
Can you add a description of what the tuple is in the docstring to make it clear in this module?  I assume class_name, class_alias, and whether it's a pipeline or not?